pkgs/by-name/fw/fw-fanctrl/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  python3Packages,
+  python3,
+  fw-ectool,
+  fetchFromGitHub,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "fw-fanctrl";
+  version = "1.0.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "TamtamHero";
+    repo = "fw-fanctrl";
+    tag = "v${version}";
+    hash = "sha256-TDVULNb/oH66/UX20mC89NSx8YPe8mPwNCB9+phavP4=";
+  };
+
+  nativeBuildInputs = [
+    python3
+  ];
+
+  dependencies = with python3Packages; [
+    fw-ectool
+    setuptools
+    jsonschema
+  ];
+
+  postInstall = ''
+    mkdir -p $out/share/fw-fanctrl
+    install $src/src/fw_fanctrl/_resources/config.json $out/share/fw-fanctrl/config.json
+    install $src/services/system-sleep/fw-fanctrl-suspend $out/share/fw-fanctrl/fw-fanctrl-suspend
```

```suggestion
    install -m 644 $src/src/fw_fanctrl/_resources/config.json $out/share/fw-fanctrl/config.json
    install -m 755 $src/services/system-sleep/fw-fanctrl-suspend $out/share/fw-fanctrl/fw-fanctrl-suspend
```

pretty sure that `config.json` shouldn't be executable